Packs/RSTCloud/Integrations/RSTCloudThreatFeedAPI/README.md

```diff
@@ -0,0 +1,11 @@
+Please contact the RST Cloud team via email support@rstcloud.net to obtain a key and ask any questions you have.
+Also, the following contact details can be used:
+- **URL**: [https://www.rstcloud.net/contact](https://www.rstcloud.net/contact)
+
+Each indicator is ranked from 0 to 100. Indicators are being collected from multiple sources and are cross-verified using multiple criteria. 
+Please check indicator tags and malware family field. An indicator may describe you a known malware or a scanning host. Threfore, different actions may be required based on the context.
+
+Usually, to exlude False Positives use the following thresholds:
+ - for IPs you start with score=50
+ - for Domains you start with score=40
+ - for URLs you start with score=30
```

Comment:
Can you please add to the readme the standard documentation by running `demisto-sdk generate-docs` command? You should run it with a `.txt` file containing command executions examples.

see `demisto-sdk generate-docs -h` for more help.

Reply:
thanks. updated.